deepspeed/runtime/comm/ccl.py

```diff
@@ -0,0 +1,137 @@
+# Copyright (c) Microsoft Corporation.
+# SPDX-License-Identifier: Apache-2.0
+
+# DeepSpeed Team
+
+import numpy as np
+import torch
+# import torch_npu
+import deepspeed.comm as dist
+from deepspeed.accelerator import get_accelerator
+from deepspeed.ops.op_builder.xpu import PackbitsBuilder
+
+
+class CCLBackend(object):
```

Comment:
From the implementation, this backend should be looked as a generic backend rather than CCL specific.  Could there be better name?